biome.json

```diff
@@ -1,9 +1,18 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "$schema": "https://biomejs.dev/schemas/2.0.5/schema.json",
```

Comment:
note: Upgraded to schema v2 by following the [v3 Upgrade Gudie](https://biomejs.dev/guides/upgrade-to-biome-v2/) and running:

```bash
$ npx @biomejs/biome migrate --write
```